### PR TITLE
Implement projects/projectName/commits endpoint

### PR DESCRIPTION
**Pull request by PMooreWand**
Implementing the projects/projectName/commits endpoint
and it's get and includedIn methods

Added tests for these parsers and api clients

### CODE REVIEW

File: src/main/java/com/urswolfer/gerrit/client/rest/http/projects/CommitApiRestClient.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @Override
+    public CommitInfo get() throws RestApiException {
+        JsonElement jsonElement = gerritRestClient.getRequest(commitURL());
+        return projectCommitInfoParser.parseSingleCommitInfo(jsonElement);
+    }
+
+
+    @Override
+    public IncludedInInfo includedIn() throws RestApiException {
+        JsonElement jsonElement = gerritRestClient.getRequest(commitURL()+"/in");
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/projects/CommitApiRestClient.java:54]: Could you please format the new code using IntelliJ default style (this and other files as well, but please only lines touched in the PR)? (this line is missing spaces before and after +)

**Review by uwolfer — COMMENTED** [review]
Thanks for this PR! Looks good, only added one minor comment. Have not tested it yet. Have you tested all the new API also against a real Gerrit server?

**Comment by PMooreWand** [comment]
> Thanks for this PR! Looks good, only added one minor comment. Have not tested it yet. Have you tested all the new API also against a real Gerrit server?

Thanks, yes I have tested it against a real gerrit myself, I maintain a fork of this project in house, incase we run into not implemented methods. This has been run against 2.16.28. Have updated the review with default formatting apologies for that.

**Comment by PMooreWand** [comment]
@uwolfer is it possible to have a release generated to include this?

**Comment by uwolfer** [comment]
@PMooreWand Sure - I have just released 0.9.6.

**Comment by PMooreWand** [comment]
Thank you, it's very much appreciated.

**Pull request merged**